gen/build_deploy/powershell/dcos_install.ps1

```diff
@@ -225,41 +236,57 @@ function Add-EnvPath {
     }
 }
 
+function SetupPathJson {
+    $jsonDoc = [pscustomobject]@{
+        install = "$($basedir)"
+        var = "$($vardir)"
+    }
+    $pathsJson = "C:\d2iq\dcos\etc\paths.json"
+    if (-not (test-path "$($pathsJson)") ) {
+        Write-Log("$($pathsJson) doesn't exist, creating it")
+    }
+    else {
+        Remove-Item -Force $pathsJson
+    }
```

Comment:
We're going to need this to be created atomically, so that existing services do not break if the file is missing or partially written. Is there a way to do this in PowerShell?